com.unity.cinemachine/Runtime/Behaviours/CinemachineFreeLookModifier.cs

```diff
@@ -42,6 +42,15 @@ public interface IModifiableScreenPosition
         Vector2 Screen { get; set; }
     }
 
+
+    /// <summary>
+    /// Interface for CinemachineComponentBase-derived to allow its bias position to be driven
+    /// </summary>
+    public interface IModifiableBiasPosition
+    {
+        Vector2 Bias { get; set; }
+    }
```

Comment:
Maybe add a composer/framingTransposer modifiable that has screen bias soft- and hard zone in one?

Reply:
I was thinking that too.  Actually, put the whole screen guides thingy in there, not just ScreenPos and Bias.  Can do it in a separate PR after this one lands.

Reply:
I'll do separately [CMCL-987](https://jira.unity3d.com/browse/CMCL-987)